notebooks/pandeia/pandeia.ipynb

```diff
@@ -1033,7 +1033,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "new_pandeia",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
Double check the name of the kernel with Todd. I think it's "romancal" or "romancalibration" or something like that.

Reply:
That is the name of the environment created to run the notebook. It is going to change depending on the users choice. In NEXUS this is handled in the instructions provided by the  first cell of the notebooks.